.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: node_js
 node_js:
   - lts/dubnium
+  - lts/erbium
```

Comment:
```suggestion
```

Reply:
I thought that more extensive testing is for the better.

Reply:
Also makes tests slower and more brittle (false positives).
We had a discussion about that for unified a while back and decided to only do the outliers.